llvm/include/llvm/ProfileData/SampleProfReader.h

```diff
@@ -494,6 +511,11 @@ class SampleProfileReader {
 
   void setModule(const Module *Mod) { M = Mod; }
 
+  void setFuncNameToProfNameMap(
+      HashKeyMap<std::unordered_map, FunctionId, FunctionId> *FPMap) {
```

Comment:
nit: use reference instead of pointer for `FPMap`, so we don't need to check/assert non-null.